package.json

```diff
@@ -9,6 +9,8 @@
     "build": "next build",
     "start": "next start -p 4444",
     "lint": "next lint",
+    "lint:fix": "next lint --fix",
+    "prepare": "node -e \"if (!process.env.CI) { require('child_process').execSync('npx simple-git-hooks', {stdio: 'inherit'}) }\"",
```

Comment:
The added script ensures that `prepare` will not run in CI, if this doesn't work, we can also set an env variable `SKIP_INSTALL_SIMPLE_GIT_HOOKS=1` to disable simple-git-hooks. For more info, see https://github.com/toplenboren/simple-git-hooks?tab=readme-ov-file#common-issues